test/unittests/clients/test_ontology_create_client_live.py

```diff
@@ -27,7 +27,8 @@
 from test.unittests.clients.constants import PROJECT_IRI
 
 ONTO = Namespace("http://0.0.0.0:3333/ontology/9999/onto/v2#")
-ONTO_IRI = URIRef("http://0.0.0.0:3333/ontology/9999/onto/v2")
+ONTO_IRI_STR = URIRef("http://0.0.0.0:3333/ontology/9999/onto/v2")
+ONTO_IRI = URIRef(ONTO_IRI_STR)
```

Comment:
Why do you create a URIRef of a URIRef?
ONTO_IRI_STR is already an URIRef, although it's called STR.